### PR TITLE
Add a python quiz question based on lambda function

### PR DESCRIPTION
**Pull request by prashalruchiranga**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1246,6 +1246,15 @@ const pythonQuiz = [
     Explanation: "{} creates an empty dictionary while set() creates an empty set.", 
     Link: "https://www.freecodecamp.org/news/free-python-crash-course/", 
   },
+  {
+    Question: "In Python, if y = lambda x : x ** 2 + 7 , then what will be the result returned when the following code is executed: print(y(3))?",
+    Answer: "16",
+    Distractor1: "5",
+    Distractor2: "11",
+    Distractor3: "24",
+    Explanation: "Lambda is an anynomous function that can have any number of arguments but only one expression, which is evaluated and returned.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1255]:
```suggestion
    Explanation: "A Lambda is an anonymous function that can have any number of arguments but only one expression, which is evaluated and returned.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/python-quiz.ts (hunk 1 above)
Comment (prashalruchiranga) [reply src/data/python-quiz.ts]: I have added the requested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**